### DISCUSSION TITLE
On the Overview Page:  Fatal error: Uncaught TypeError

### DISCUSSION THREAD

**Discussion by MrJasonBryant** [rdz-oss/BattyBirdNET-Pi#51] [General]
Any idea how to fix or resolve these issues? Fatal error: Uncaught TypeError: array_map(): Argument #2 ($array) must be of type array, bool given in /home/birder/BirdNET-Pi/scripts/overview.php:161 Stack trace: #0 /home/birder/BirdNET-Pi/scripts/overview.php(161): array_map() #1 {main} thrown in /home/birder/BirdNET-Pi/scripts/overview.php on line 161

5 Most Recent Detections

Fatal error: Uncaught TypeError: array_map(): Argument #2 ($array) must be of type array, bool given in /home/birder/BirdNET-Pi/scripts/todays_detections.php:288 Stack trace: #0 /home/birder/BirdNET-Pi/scripts/todays_detections.php(288): array_map() #1 {main} thrown in /home/birder/BirdNET-Pi/scripts/todays_detections.php on line 288

**Comment by EvaldoOliveira** [comment]
Clean database and then try creating an empty exception file, or add one specie to it, later remove it.
This worked for me.
Looks like this happens when the file does not exists

**Comment by MrJasonBryant** [comment]
thanks! ill give that a try

**Comment by MrJasonBryant** [comment]
you my friend are a godsend! everything is up and working as expected. :) thanks again!
